Formula/libmypaint.rb

```diff
@@ -0,0 +1,31 @@
+class Libmypaint < Formula
+  desc "MyPaint brush engine library"
+  homepage "https://github.com/json-c/json-c/wiki"
```

Comment:
i've replaced link to https://github.com/mypaint/libmypaint/wiki because http;//mypaint.org contains too little information about libmypaint